src/BookDetails/BookDetails.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+export const BookDetails = ({book}) => {
+
+    return(
+        <section>
+          <button>Close Details</button>
+          <article>
+            <img src={book.image}/>
+            <h2>{book.title}</h2>
+            <p>{book.summary}</p>
+          </article>
+          <article>
+            <div>PROGRESS BAR</div>
+            <button>Add A Bookmark</button>
+            <button>Finish Book</button>
+            <button>Abandon Book</button>
```

Comment:
Do we want a form/alert in response to the `Abandon` button? E.g. "Are you sure?" or "Why didn't this one work out?"

Reply:
I think that's a great idea!